README.md

```diff
@@ -16,7 +16,10 @@ the square brackets (for multiple-choice questions).
 What does it mean for an HTML tag to be "semantic"? Give an example of a short snippet of HTML written using semantic tags and non-semantic tags.
 
 ```text
-Your answer here.
+Semantic means that the whatever it is has meaning. It is not the style but the actual content.
+
+<p1>Welcome to my page</p1> - This is semantic.
+font-size: 20px; - This is non-semantic
```

Comment:
✅  I'll give it to you, but `p1` is an HTML tag you invented, rather than one of the standard ones. Stick to the standard ones.